index.html

```diff
@@ -126,98 +126,8 @@
                 <span class="line"></span>
             </p>
 
-            <div class="speakers">
+            <div class="speakers" id="speakers">
```

Comment:
- [ ] Good job with your navbar but I think you can move your menu inside `nav` tag for the semantic reason also please work on your design in order to make it similar to the original template :+1: 

### Original design
![Screenshot from 2022-10-19 20-04-46](https://user-images.githubusercontent.com/6615394/196793670-bdd829a7-a34e-4f9f-9d89-eb684f2fa757.png)

### Your design
![Screenshot from 2022-10-19 20-04-23](https://user-images.githubusercontent.com/6615394/196793685-0a345728-055c-4535-a878-3b8ec373b211.png)